server/channels/store/sqlstore/channel_store.go

```diff
@@ -1935,6 +1935,12 @@ func (s SqlChannelStore) PatchMultipleMembersNotifyProps(members []*model.Channe
 		return nil, err
 	}
 
+	transaction, err := s.GetMasterX().Beginx()
```

Comment:
nit: I /think/ it makes sense to defer `Beginx` until as late as possible, vs. pulling it up? (not sure it has to be at the top of the method, for example.)

Reply:
I can change that back. I pulled it up because I thought `transaction.ExecBuilder` was more like `s.getQueryBuilder` where it returns the builder, and I forgot to change it back